payload/dev/docker-compose.yml

```diff
@@ -30,6 +30,9 @@ services:
             - PROJECTMAPPINGFOLDER
             - DEV_UID
             - DEV_GID
+            - XDEBUG_ENABLED
+            - "XDEBUG_CONFIG=remote_host=172.17.0.1"
```

Comment:
Is that generic for Mac OS and Linux?

Reply:
Sorry haven't seen your comment, gonna look!